src/balatro_gym/cards/interfaces.py

```diff
@@ -375,8 +375,8 @@ def set_seal(self, seal: Seal) -> None:
     def add_chips(self, num_chips: int) -> None:
         self._added_chips += num_chips
 
-    def is_face_card(self) -> bool:
-        return self._rank in [Rank.KING, Rank.QUEEN, Rank.JACK]
+    def is_face_card(self, has_pareidolia: bool) -> bool:
```

Comment:
What do you think about passing in `BoardState` here instead of `has_pareidolia` and extract `has_pareidolia` from `board_state.jokers ? It would follow the pattern of other functions-- basically with the expectation that the special cases can be contained within a class and not leak into the client code

Reply:
That's originally what I did (passed the jokers), but it's causing circular dependencies, because we're calling the joker with typing classes . I tried to move a bunch of things around (`PlayingCard`,`Rank`, `Suit`), but it wasn't working at all, so I settled on this (which I don't love)